os/arch/arm/src/armv8-m/up_assertlogs.c

```diff
@@ -0,0 +1,570 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
```

Comment:
If this is a new file, could you change the year to 2022 in the header signature please?